### PR TITLE
Program for conversion SSD models into intel-i8 format

### PR DESCRIPTION
**Pull request by Chunosov**

### CODE REVIEW

File: program/convert-ssd-to-i8/README.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+```
+ck install package --tags=lib,caffe,intel
+```
+
+Caffe model to be converted into quantized format:
+```
+ck install package --tags=caffemodel,ssd
+```
+
+One of datasets COCO-2014 or VOC-2207 depending on wich model you want convert:
```
Comment (psyhtest) [inline program/convert-ssd-to-i8/README.md]: VOC-2007?

Comment (psyhtest) [inline program/convert-ssd-to-i8/README.md]: depending on which model you wish to convert:

File: program/convert-ssd-to-i8/README.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+Resulting models are stored into `tmp` directory as `test_quantized.prototxt` and `deploy_quantized.prototxt`.
+
+## Test
+
+Program also provide two testing commands:
+```
+ck run program:convert-ssd-to-i8 --cmd_key=test-f32
+ck run program:convert-ssd-to-i8 --cmd_key=test-i8
+```
+Commands take LMDB database that was prepared at previous `convert` stage and run `caffe test` operaion using original or converted models.
```
Comment (psyhtest) [inline program/convert-ssd-to-i8/README.md:44]: `caffe test` -> `ck run program:caffe --test_cpu` ?

**Comment by psyhtest** [comment]
Does this handle both `300x300` and `512x512` SSD variants?

File: program/convert-ssd-to-i8/README.md (hunk 1 above)
Comment (Chunosov) [reply program/convert-ssd-to-i8/README.md]: Fixed

Comment (Chunosov) [reply program/convert-ssd-to-i8/README.md]: Changed.

File: program/convert-ssd-to-i8/README.md (hunk 2 above)
Comment (Chunosov) [reply program/convert-ssd-to-i8/README.md:44]: No, it does not run `ck run program:caffe`, it runs
```
${CK-TOOLS}/lib-caffe-*/install/bin/caffe test --model=... --weights=... --iterations=... --detect
```

**Comment by Chunosov** [comment]
Yes it does, it is only a matter of LMDB preparation. 
But conversion of 512-px model is quite slower, I think because of additional layers that this model has.
Actually I have not tried it with COCO as it takes about 7Gb to install its dataset, but only about 10 is free on velociti. But models are the same for COCO and VOC.

**Comment by psyhtest** [comment]
>  But models are the same for COCO and VOC.

I believe the models are slightly different because classify into different number of classes...

**Pull request merged**